_posts/blog/2016-10-05-dwh-01.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "BTW, we need a DWH. Yesterday!"
+description: The beginning of our journey towards a custom DWH.
+modified:
+categories: blog
+author: robert_bordo
+excerpt: The first part of a series of articles chronicling our fellow data platform team on their quest for a functional custom data warehouse solution.
+tags: [dwh, data-warehouse]
+date: 2016-10-05T10:00:00-04:00
+---
+
+_Data is the new bacon. Or is it the new oil? Maybe gold? Regardless, it seems to be valuable. This is why we need our own data warehouse (DWH). Now!_
+
+At upday we’ve __collected anonymous user data since the beginning__, in order to __improve user experience__ — particularly content recommendations. For example, we categorise articles and track the topics that users read about, then with some fancy algorithms we try to predict and satisfy our users but also strive to surprise them with articles they could be interested in (we call it serendipity). 
+
+Another use case for collected data is to __understand how our users behave within the app__. You can do field research, engage test audiences and ask friends, but in reality the user always acts differently. And very often the differences are surprising.
+
+Finally, even a cool startup like upday needs to __generate revenue__ someday. Since we originate from a media company, we have a strong motivation to support journalism. This is why we also need to know which articles have been read and how often, in order to __share revenue with the original content publishers__. Unsurprisingly, that the publishers we cooperate with have a similar opinion.
```

Comment:
for me this:
> Finally, even a cool startup like upday needs to __generate revenue__ someday


sounds like mocking. It's a bit too ironic. 
I would remove the sentence completely since you include the revenue part.